packages/medusa/src/api/routes/admin/inventory-items/delete-inventory-item.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from "express"
+import { IInventoryService } from "../../../../interfaces"
+
+/**
+ * @oas [delete] /inventory-itemss/{id}
+ * operationId: "DeleteInventoryItemsInventoryItem"
+ * summary: "Delete an Inventory Item"
+ * description: "Delete an Inventory Item"
+ * x-authenticated: true
+ * parameters:
+ *   - (path) id=* {string} The ID of the Inventory Item to delete.
+ * x-codeSamples:
+ *   - lang: JavaScript
+ *     label: JS Client
+ *     source: |
+ *       import Medusa from "@medusajs/medusa-js"
+ *       const medusa = new Medusa({ baseUrl: MEDUSA_BACKEND_URL, maxRetries: 3 })
+ *       // must be previously logged in or use api token
+ *       medusa.admin.inventoryItems.delete(inventory_item_id)
```

Comment:
suggestion(non blocking): In other places we are naming `inventoryItemId` and here it is `inventory_item_id`